test/unit/endpoint-client.test.ts

```diff
@@ -332,4 +337,21 @@ describe('Endpoint Client',  () => {
 		expect(threwError).toBe(true)
 		axios.request.mockReset()
 	})
+
+	it('does not log bearer token in request', async () => {
+		axios.request.mockImplementationOnce(() => Promise.resolve({ status: 200, data: { status: 'ok' } }))
+		const bearerToken = '00000000-0000-0000-0000-000000000000'
+		const config: EndpointClientConfig = {
+			authenticator: new BearerTokenAuthenticator(bearerToken),
+			logger: new NoLogLogger,
+		}
+		jest.spyOn(NoLogLogger.prototype, 'isDebugEnabled').mockReturnValue(true)
+		const debugSpy = jest.spyOn(NoLogLogger.prototype, 'debug')
+		const bearerClient = new EndpointClient('basePath', config)
+
+		await bearerClient.get('')
+
+		expect(debugSpy).toBeCalled()
+		expect(debugSpy).not.toBeCalledWith(expect.stringContaining(bearerToken))
```

Comment:
I think it would be useful here to have a `expect` for at least part of the truncated message (e.g. `Bearer 00`) just to make sure that something didn't get broken that causes it to not include anything at all, particularly if that something is inside the test itself which would mean the test isn't testing anything.

Reply:
Good call. I made this change and also added a second test for the backup redact with a similar approach.